test/test_timeseries.py

```diff
@@ -477,6 +477,33 @@ def test_duration(self):
             self.assertAlmostEqual(self.b.duration, 0.3)
             self.assertAlmostEqual(self.bad3.duration, 0.6)
 
+    def test_at_time(self):
```

Comment:
I don't really know how our github tests work, will this be run automatically everytime a commit is pushed?

Reply:
Yes, that's right. These are all run automatically.